tests/test_generator.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+import numpy as np
+
```

Comment:
Again, It could be good to at short description what this test should check ?

Reply:
I added some comments to the test.